Source/OIDTokenUtilities.m

```diff
@@ -48,4 +48,8 @@ + (NSData *)sha265:(NSString *)inputString {
   return sha256Verifier;
 }
 
++ (NSString *)truncate:(NSString *)inputString {
+    return [inputString length] >= 6 ? [[inputString substringToIndex:6] stringByAppendingString:@"..."] : @"...";
```

Comment:
Bit of a nit here - but the current case of length == 6 is a bit degenerate... we probably shouldn't show the ellipsis when we aren't actually removing any characters. Also, maybe instead of the ellipsis for < the threshold we could just say "[redacted]" to be clear?

How about this.. ?

`return [inputString length] >= 9 ? [[inputString substringToIndex:6] stringByAppendingString:@"..."] : @"[redacted]";`